packages/sdk-core/src/modules/contract/index.ts

```diff
@@ -33,17 +33,26 @@ export const sendTransaction = async (api: ApiPromise, contract: ContractPromise
       ) as WeightV2,
       storageDepositLimit: null,
       value
-    }
+    },
+    ...args
   );
 
   if (result.result.isErr) {
-    throw result;
+    let display = '';
+    if (result.result.asErr.isModule) {
```

Comment:
I think this is related to the comment I made on Slack. Can you elaborate what was the cause with the arguments and how does this fix it?

Reply:
Line 37 fixes the args issue. This code was requested by mario to move the error handling in the helper function

Reply:
ahh, okay. So the function literally didn't pass any arguments 💀